modal/functions.py

```diff
@@ -306,6 +306,9 @@ class _Function(_Object, type_prefix="fu"):
     #  this references the parent class-function and is used to infer the client for lazy-loaded methods
     _parent: Optional["_Function"] = None
 
+    _class_parameter_format: Optional["api_pb2.Function.ParameterSerializationFormat.ValueType"] = None
```

Comment:
j/w but why is a forward reference for the type annotation needed here?

Reply:
I haven't fully investigated why, but there are runtime errors for the proto enum `.ValueType` in Python 3.8 (we usually just type them as `int` and add a comment, but I think this is nicer)

Reply:
Oh cool, nice to know that this works.